scripts/openmc-ace-to-hdf5

```diff
@@ -111,25 +118,25 @@ for filename in ace_libraries:
             # Continuous-energy neutron data
```

Comment:
You _could_ fix this by just removing duplicate etnries from ace_libraries.  To approximate this, I simplified my cross_sections.xml file so that I only had one table from each library file; seems to work.  

The problem with this is that if someone took care to remove data they didnt need from their xsdir/xsdata/cross_sections.xml file (like non-relevant temperatures or isotopes), then this strategy would over-rule that explicit decision the user has made.